src/index.ts

```diff
@@ -1,13 +1,17 @@
 import { config } from 'dotenv'
 config()
-import express from 'express'
+import express, { Request, Response } from 'express'
 import * as OpenApiValidator from 'express-openapi-validator'
+import { elasticHealthCheck } from './config/elasticsearch.config'
+import { ErrorResponseType } from './interfaces/common.interface'
 import appRoutes from './routes/index'
 
 const app = express()
 
 app.use(express.json())
 
+elasticHealthCheck()
```

Comment:
Just a suggestion, no need to add it now but do give it a read. Dor health check on an application we can use actuator, dependency: express-actuator